src/frequenz/sdk/actor/power_distributing/_battery_status.py

```diff
@@ -5,6 +5,7 @@
 from __future__ import annotations
 
 import asyncio
+import datetime as dt
```

Comment:
Please don't use the `dt` alias to keep a consistent style in the SDK (we usually don't use it in the SDK).
```suggestion
import datetime
```